io_scene_nif/modules/nif_export/property/object/__init__.py

```diff
@@ -50,6 +52,9 @@
 
 
 class ObjectProperty:
```

Comment:
I would nearly rename this class as MaterialPropertyProcessor ?

Reply:
Well, not really, since it processes object properties as well (wireframe). But I would perhaps integrate the MaterialProp class into ObjectProperty, since it has just a single function.

Reply:
I was more thinking that Object processing would be more for the likes of processing Object level properties like flags, extradata, etc. But its probably something for down the road.